### ISSUE TITLE
It's not available to fragment? 

### ISSUE THREAD

**Issue by SuperMrHe** [TakuSemba/MultiSnapRecyclerView#8]
It's alway report "other than LinearLayoutManger is not supported"

**Comment by TakuSemba** [comment]
I think It's available to fragment. 
What type of layout manager are you using ??

**Comment by tsuyosh** [comment]
I also had a similar issue. The stacktrace is following.
I solved the problem with calling setLayoutManager() at Fragment#onCreateView() NOT at Fragment#onViewCreated().

```
java.lang.IllegalArgumentException: other than LinearLayoutManger is not supported
    at com.takusemba.multisnaprecyclerview.MultiSnapRecyclerView.onAttachedToWindow(MultiSnapRecyclerView.java:41)
    at android.view.View.dispatchAttachedToWindow(View.java:14520)
    at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:2836)
    at android.view.ViewGroup.dispatchAttachedToWindow(ViewGroup.java:2843)
    at android.view.ViewGroup.addViewInner(ViewGroup.java:4348)
    at android.view.ViewGroup.addView(ViewGroup.java:4145)
    at android.view.ViewGroup.addView(ViewGroup.java:4086)
    at android.view.ViewGroup.addView(ViewGroup.java:4059)
    at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1425)
    at android.support.v4.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1750)
    at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1819)
    at android.support.v4.app.BackStackRecord.executeOps(BackStackRecord.java:797)
    at android.support.v4.app.FragmentManagerImpl.executeOps(FragmentManager.java:2590)
    at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2377)
    at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2332)
    at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2239)
    at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:700)
    at android.os.Handler.handleCallback(Handler.java:739)
    at android.os.Handler.dispatchMessage(Handler.java:95)
    at android.os.Looper.loop(Looper.java:148)
    at android.app.ActivityThread.main(ActivityThread.java:5417)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)

```

**Comment by TakuSemba** [comment]
thank you for reporting, I understand.
I will try to fix it.

**Comment by onooma** [comment]
I have the same problem, the first time it's ok but when I replaced the fragment again app crashed.

**Comment by TakuSemba** [comment]
this has just fixed at version 1.1.1 👍
